src/Frontend/Modules/Search/Layout/Sass/Search.scss

```diff
@@ -0,0 +1,206 @@
+@import 'bootstrap-sass/assets/stylesheets/bootstrap/variables';
+
+// a mixin to simplify our code
+@mixin typeahead-form-control($border-color) {
+  border-color: $border-color;
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+
+  &:focus {
+    border-color: darken($border-color, 10%);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px lighten($border-color, 20%);
+  }
+}
+
+// coloring based on state
+.has-warning .twitter-typeahead {
+  .tt-input, .tt-hint {
+    @include typeahead-form-control($state-warning-text);
```

Comment:
I used the same variables as the less-version. And I think they use these vars, because the focus-state calculates it value based on this variable